.gitignore

```diff
@@ -4,5 +4,8 @@ public/lib
 .DS_store
 c4h_keys.txt
 .env
+npm-debug.log
+.package-lock.json
```

Comment:
I'm curious to know why this is included twice?

Reply:
that was done in error and have been corrected.